### PR TITLE
chore: add functional tests

### LINKED ISSUE #3: Add functional tests
We should add some functional tests.

Possibly with the travis rabbitmq service

### PR DESCRIPTION
**Pull request by cawolf**
fixes #3

### CODE REVIEW

File: Tests/Functional/TestProjectFiles/default/src/Consumer/TestConsumer.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    public function __construct(Tracing $tracing)
+    {
+        $this->tracing = $tracing;
+    }
+
+    public function execute(AMQPMessage $msg)
+    {
+        $this->tracing->startActiveSpan('manual consumer span');
+        $this->tracing->setTagOfActiveSpan('test.tag', 'manual');
+        usleep(10000);
```
Comment (MollocH) [inline Tests/Functional/TestProjectFiles/default/src/Consumer/TestConsumer.php:24]: This is related to the fact that a span must have a minimum duration, right ?

**Review by MollocH — APPROVED** [review]

**Comment by MollocH** [comment]
LGTM

File: Tests/Functional/TestProjectFiles/default/src/Consumer/TestConsumer.php (hunk 1 above)
Comment (cawolf) [reply Tests/Functional/TestProjectFiles/default/src/Consumer/TestConsumer.php:24]: No, just for visualizing a heavy computation

**Pull request merged**